lumigator/python/mzai/schemas/lumigator_schemas/jobs.py

```diff
@@ -10,6 +10,7 @@ class JobType(str, Enum):
     INFERENCE = "inference"
     EVALUATION = "evaluate"
     EVALUATION_LITE = "eval_lite"
+    ANNOTATION = "annotate"
```

Comment:
❓ Why do we not have this mapped to the string `annotation`? 

I see above that there's weird conflicting styles with `inference` vs `evaluate` (instead of `evaluation`) if we're choosing one way over the other, should we just update these to all be the same?

Reply:
Totally true. These strings correspond to the API endpoints. We should definitely use a single category (verbs vs nouns) at some point.